### PR TITLE
send-http-request added

### PR DESCRIPTION
**Pull request by SanjayDevTech**
Fixes: [https://github.com/appwrite/appwrite/issues/4181](https://github.com/appwrite/appwrite/issues/4181)

- Implemented function `sendHTTPRequest()` in `kotlin/` folder.
- Manually executed the newly implemented function to ensure it works using openruntime docker image.

### CODE REVIEW

**Comment by abnegate** [comment]
@SanjayDevTech Thanks for the PR 🤯 Please give us some time to review it 🙏

**Comment by Meldiron** [comment]
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

File: kotlin/send-http-request/Index.kt [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                ?: return res.json(mapOf("success" to false, "message" to "Invalid json data"))
+        val headers: Map<String, String> =
+            gson.fromJson<Map<String, String>>(json["headers"].toString(), MutableMap::class.java)
+                ?: mapOf()
+        val payload =
+            Payload(
+                method = json["method"].toString(),
+                url = json["url"].toString(),
+                headers = headers,
+                body = if (json["method"].toString() != "GET") json["body"].toString() else null,
```
Comment (stnguyen90) [inline kotlin/send-http-request/Index.kt:52]: I tried testing with:

```json
{"headers": {"Content-Type": "application/json"},"url": "https://jsonplaceholder.typicode.com/posts", "method": "POST", "body": "{\"title\": \"foo\"}"}
```

and

```json
{"headers": {"Content-Type": "application/json"},"url": "https://jsonplaceholder.typicode.com/posts", "method": "POST", "body": {"title": "foo"}}
```

but I'm getting the following error: 

```json
{"success":false,"message":"JSON payload parsing error: com.google.gson.stream.MalformedJsonException: Unterminated object at line 1 column 27 path $."}
```

Any idea what could be wrong?

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Great PR! 🤯 We left some comments during the review, please check them out.

Would you also please add a README like https://github.com/open-runtimes/examples/blob/1ec5fc25cd5ec71615d44b4ae33214e1d320d25b/kotlin/deepgram-transcribe-audio/README.md?

**Comment by Meldiron** [comment]
Hey there 👋
We are temporarily closing your PR as there has been no activity on it. Feel free to re-open it when you're ready to start working on it again 😇

**Comment by SanjayDevTech** [comment]
@stnguyen90 Just want to know how to test the functions immediately after the code change?

**Comment by SanjayDevTech** [comment]
@Meldiron Can you re-open this pr. I can't see any option to reopen. Thanks

**Pull request closed without merging**